### PR TITLE
Resource Graph:  adding param to extend query result with subscription and tenant names

### PR DESCRIPTION
**Pull request by OlhaTkachenko**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/ResourceGraph/ResourceGraph/ResourceGraph.csproj [hunk 1]
```diff
@@ -14,4 +14,8 @@
     <PackageReference Include="Microsoft.Azure.Management.ResourceGraph" Version="0.10.0-preview" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\Accounts\Accounts\Accounts.csproj" />
```
Comment (cormacpayne) [inline src/ResourceGraph/ResourceGraph/ResourceGraph.csproj]: @OlhaTkachenko this reference should be removed

File: src/ResourceGraph/ResourceGraph/Cmdlets/SearchAzureRmGraph.cs [hunk 2]
```diff
@@ -186,5 +238,32 @@ private IEnumerable<string> GetSubscriptions()
...
+
+        /// <summary>
+        /// Gets list of detailed tenants 
+        /// </summary>
+        /// <returns></returns>
+        private IList<IDictionary<string, object>> ListTenants()
+        {
+            try
+            {
+                var profileClient = new RMProfileClient(AzureRmProfileProvider.Instance.GetProfile<AzureRmProfile>());
```
Comment (cormacpayne) [inline src/ResourceGraph/ResourceGraph/Cmdlets/SearchAzureRmGraph.cs]: @OlhaTkachenko you should not be referencing types from the Accounts project -- all authentication needs to be encapsulated inside of the Accounts project and not used by any other project. What is the goal for the below code?

Comment (OlhaTkachenko) [reply src/ResourceGraph/ResourceGraph/Cmdlets/SearchAzureRmGraph.cs]: We are doing ARM call not using SDK. It is why we need to get authorization token. Please let me know if you have other ideas. I will add tag, maybe somebody will know

**Comment by markcowl** [comment]
@OlhaTkachenko You will need to submit a cmdlet review for this, here:

We need to discuss what the intended change is, and why you need to use types protected in Az.Accounts

**Comment by cormacpayne** [comment]
@OlhaTkachenko we should be able to continue with this PR once the PR containing the updated Subscriptions client is merged: https://github.com/Azure/azure-powershell/pull/9647

**Comment by markcowl** [comment]
@OlhaTkachenko #9647 is merged, please update this PR to consume the new library

File: src/ResourceGraph/ResourceGraph/ResourceGraph.csproj (hunk 1 above)
Comment (OlhaTkachenko) [reply src/ResourceGraph/ResourceGraph/ResourceGraph.csproj]: @cormacpayne , thank you for updating ARM client! I removed this reference.

**Comment by OlhaTkachenko** [comment]
> @OlhaTkachenko You will need to submit a cmdlet review for this, here:https://github.com/azure/azure-powershell-cmdlet-review-pr/issues
> 
> We need to discuss what the intended change is, and why you need to use types protected in Az.Accounts

Removed the reference. Do I still need to submit this review?

File: src/ResourceGraph/ResourceGraph.Test/ScenarioTests/ResourceGraphQueryTests.cs [hunk 3]
```diff
@@ -50,7 +50,14 @@ public void Subscriptions()
         {
             TestController.NewInstance.RunPowerShellTest(_logger, "Search-AzureRmGraph-Subscriptions");
         }
-        
+
+        [Fact(Skip = "Needs investigation, TestManagementClientHelper class wasn't initialized with the ResourceManagementClient client.")]
```
Comment (cormacpayne) [inline src/ResourceGraph/ResourceGraph.Test/ScenarioTests/ResourceGraphQueryTests.cs]: @OlhaTkachenko this error means that the `ResourceManagementClient` initialized in your `TestController` class is different (either by the package it comes from or the package version) than the `ResourceManagementClient` initialized in your cmdlet code. With the recent changes you made, I'm not sure if you're using the `ResourceManagementClient` anymore in your cmdlet code, so if you aren't, you should be able to run this test again.

Comment (OlhaTkachenko) [reply src/ResourceGraph/ResourceGraph.Test/ScenarioTests/ResourceGraphQueryTests.cs]: done

**Pull request merged**